tools/clang/lib/SPIRV/SPIRVEmitter.cpp

```diff
@@ -1293,8 +1293,9 @@ uint32_t SPIRVEmitter::doUnaryOperator(const UnaryOperator *expr) {
     uint32_t incValue = 0;
     if (TypeTranslator::isSpirvAcceptableMatrixType(subType)) {
       // For matrices, we can only increment/decrement each vector of it.
-      const auto actOnEachVec = [this, spvOp, one](
-          uint32_t /*index*/, uint32_t vecType, uint32_t lhsVec) {
+      const auto actOnEachVec = [this, spvOp, one](uint32_t /*index*/,
```

Comment:
Somehow we compete to reformat this code snippet. Maybe because we use different versions of clang-format. ;-P